src/Symfony/Component/Dotenv/Dotenv.php

```diff
@@ -271,9 +271,6 @@ private function lexValue()
                         throw $this->createFormatException('Missing quote to end the value');
                     }
                 }
-                if ("\n" === $this->data[$this->cursor]) {
```

Comment:
Are you sure that this can never be reached?

Reply:
Yes, on line 266 while loop executes until `$this->data[$this->cursor]` is equal to `"`, and if cursor reaches the end - throws `FormatException`. So, `"\n" === $this->data[$this->cursor]` will always resolve to false.